crates/matrix-sdk-base/src/client.rs

```diff
@@ -460,7 +460,11 @@ impl BaseClient {
         let mut user_ids = BTreeSet::new();
         let mut profiles = BTreeMap::new();
 
+        assert_eq!(raw_events.len(), events.len());
         for (raw_event, event) in iter::zip(raw_events, events) {
+            let Some(event) = event else {  continue;
+            };
```

Comment:
Is this the doing of `cargo fmt`? :scream:

Reply:
Ah nope, it seems incorrect; you might need an extra `cargo fmt` run.

Reply:
Oops, I did not mean to push that. Actually inserted some bad whitespace to make sure cargo fmt ran, but I can't seem to get it to run for some reason.

Reply:
Ah, I needed `rustup run nightly cargo fmt`, fixed!